docs/getting-started/installation/containers.md

```diff
@@ -4,123 +4,41 @@ sidebar_position: 4
 
 # Containers
 
-**Installing Forem using Containers**
+We offer two options for container-based development.
 
-If you encounter any errors with our Container setup, please kindly
-[report any issues](https://github.com/forem/forem/issues/new/choose)!
-
-## Installing prerequisites
-
-_These prerequisites assume you're working on an operating system supported by
-Docker or Podman._
-
-### Choosing a Container Engine
-
-A container engine is software that runs and manages containers on a computer.
-One of the most widely known Container Engines is Docker, but there are many
-other Container Engines available, such as [Podman](https://podman.io/),
-[CRI-O](https://cri-o.io/), and
-[LXD](https://linuxcontainers.org/lxd/introduction/).
-
-Forem supports two Container Engines: Docker and Podman.
-
-### Docker
-
-Forem can be setup with Docker and Docker Compose on macOS or Linux systems.
-
-Docker is available for many different operating systems. You may use Docker as
-your Container Engine on both macOS and Linux workstations. As of right now
-Docker is the only Container Engine for macOS and we recommend you follow the
-[Docker Desktop on Mac](https://docs.docker.com/docker-for-mac/install/),
-install instructions to get Docker and Docker Compose installed.
-
-Docker also works well on Linux distributions that have not moved to cgroup v2.
-You can install it by following their
-[Installation per distro](https://docs.docker.com/engine/install/) to get Docker
-and you can install Docker Compose by following these
-[instructions](https://docs.docker.com/compose/install/).
-
-### Podman
-
-Forem can be setup with Podman and Podman Compose on Linux systems.
+1. VS Code with Devcontainer
+1. Containerized setup with `dip` gem
 
-[Podman](https://podman.io/) is an FOSS project that provides a Container Engine
-that is daemonless which only runs on Linux systems. It can be run as the root
-user or as a non-privileged user. It also provides a Docker-compatible command
-line interface. Podman is available on many different Linux distributions and it
-can be installed by following these
-[instructions](https://podman.io/getting-started/installation).
+Due to the way our we setup our `docker-compose.yml`, standard `docker-compose up` command is not support. Please checkout the "Containerized setup with `dip` gem option"
```

Comment:
Not supported? (instead of "not support")

Also, not a big deal but as I understood, `docker-compose up` was replaced by [`docker compose up` ](https://docs.docker.com/engine/reference/commandline/compose_up/)

Reply:
ah right yeah forgot about that